sig-wg-lifecycle.md

```diff
@@ -19,13 +19,26 @@ Out of scope for this document: [subproject] creation.
 - [ ] Read [wg-governance.md]
 - [ ] Ensure all WG Organizers, and other leadership roles are [community members]
 - [ ] Send email to [kubernetes-dev@googlegroups.com] titled "WG-Creation-Request: WG Foo" with some of the questions answered from wg-goverance.md and wait for community discourse; ask for SIG sponsorship
-- [ ] Do the first checklist item in the #GitHub section below and add a row to the WG section:
+- [ ] Do the first checklist item in the [GitHub] section below and add a row to the WG section:
   - [ ] Label with committee/steering and wait for a simple majority
   - [ ] Also add sponsoring SIG Chair/Tech Leads as approvers; you'll get this from the community email above
   - [ ] Place a `/hold` on it until the members that need to review have; a contributor experience member will do this for you if they don't see one already
 - [ ] Send an email to the stakeholder SIG mailing lists and steering committee with the sigs.yaml pull request
 
-### GitHub:
+### Prerequisites for a UG
+- [ ] Read [ug-governance.md]
+- [ ] Require to have 2 Chairs to start a User Group.
```

Comment:
I'd prefer a vote from a sponsoring SIG imo.

Reply:
What is the Sponsoring SIG?  Do we need one?

Reply:
Per-discussion the 2 sponsors + lazy consensus sgtm

Reply:
@timothysc Who are the 2 sponsors? 2 SIG Chair sponsors? I forget.